### DISCUSSION TITLE
I have paid twice for a service and have a web reciept for both times and have gotten nothing

### DISCUSSION THREAD

**Discussion by Kstar68** [btcpayserver/btcpayserver#4879] [Plugins & integrations]
I have already paid twice for a service and have seen nothing and have web reciept for both now what do I do?

**Comment by mp21-org** [comment]
You need to contact the merchant directly. We are not a payment service. We only provide a free and open source software.
Also make sure you are not being scammed. Especially Bitcoin Bonus causes a lot of problems/damage at the moment: https://blog.btcpayserver.org/bitcoin-bonus-scam/
